src/backend/distributed/operations/shard_rebalancer.c

```diff
@@ -1014,6 +1020,12 @@ citus_rebalance_start(PG_FUNCTION_ARGS)
 	PG_ENSURE_ARGNOTNULL(2, "shard_transfer_mode");
 	Oid shardTransferModeOid = PG_GETARG_OID(2);
 
+	PG_ENSURE_ARGNOTNULL(3, "parallel_transfer_reference_tables");
```

Comment:
My preference is to keep it simple and have `one parallel flag` to do both. It's too late now, we can look at it later